docs/docsite/rst/network/dev_guide/developing_plugins_network.rst

```diff
@@ -0,0 +1,108 @@
+
+.. _developing_modules_network:
+.. _developing_plugins_network:
+
+**************************
+Network connection plugins
+**************************
+
+Each network connection plugin has a set of plugins which provide a specification of the connection for a particular set of devices. Public methods of these plugins may be called on the connection proxy object from the module just as connection methods can.
+
```

Comment:
A paragraph explaining how `Connection` knows about the proper plugin could be helpful to someone newer to the framework. I'll include a proposal.